.husky/pre-commit

```diff
@@ -1,4 +1,11 @@
-#!/bin/sh
+#!/bin/bash
 . "$(dirname "$0")/_/husky.sh"
 
-npx git-format-staged -f 'npx prettier --ignore-unknown --stdin-filepath "{}"' 'src/*.ts' 'src/*.js' 'src/*.html' 'src/*.json' 'src/*.scss'
+npx_version=$(npx --version)
+npx_version_major="${npx_version:0:1}"
+echo $npx_version_major;
```

Comment:
maybe we don't need it or a better message is welcome.

Reply:
Indeed :-) That was just for debugging...